### DISCUSSION TITLE
How to change the keyboard

### DISCUSSION THREAD

**Discussion by sventanta10** [soywod/react-pin-field#89] [General]
Good evening to all,
I'm looking to change the default keyboard offered when entering Pin from my smartphone.

Currently, a standard keyboard is proposed, but I'd like to have the numeric keyboard.

Thanks for your help :)

**Comment by soywod** [comment]
Did you try the property `type="number"`? See the [Validate](https://soywod.github.io/react-pin-field/) section of the demo, it should propose a numeric keyboard.
